### ISSUE TITLE
Cannot reorder draggable items via keyboard/screen reader

### ISSUE THREAD

**Issue by jonathanmeaney1** [Sage/carbon#7464]
### Description

Reordering items on pages that use the `DraggableContainer` and `DraggableItem` components can be done by clicking and dragging items. There is no alternative way to reorder these items using a keyboard or screen reader, meaning these actions are unable to be completed by assistive technology users.

This issue was found during an accessibility audit.

Is there anything the Carbon team can do to make the `DraggableContainer` and `DraggableItem` components more accessible via the keyboard/screen reader?

### Reproduction

https://carbon.sage.com/?path=/docs/draggable--docs#simple-text-as-a-content

### JIRA ticket numbers (Sage only)

SKPU-13721

### Carbon version

v154.0.0

### React version

v17

### Design tokens version

^4.35.0

### Relevant browsers

Chrome

### Relevant OSs

MacOS

### Confidentiality

- [x] I confirm there is no confidential or commercially sensitive information included.

**Comment by edleeks87** [comment]
The suggested pattern is to use an additional component to handle to moves when not using mouse. Here's an example using [ActionPopover](https://stackblitz.com/edit/parsium-carbon-starter-g5svyahv?file=src%2FApp.tsx). We are currently working on a rewrite of this behaviour to support more flexibility but I suspect an additional component will still be needed. I'm going to close this as I don't think it's a bug but if you have issues with the above please re-raise and we can review with DS/UX
